CHANGES.rst

```diff
@@ -44,6 +44,12 @@ ramp_fitting
 
 - Update documentation to define optimal weighting algorithm [#5682]
 
+source_catalog
```

Comment:
Should also now add an entry to the change log under `datamodels`, because of the new SEGMFILE keyword that's being added.